docs/Call_for_papers_Bioinformatics.md

```diff
@@ -0,0 +1,132 @@
+# SciPyLA 2017
+## Call for Bioinformatics Research Papers
+
+![Alt text](../logo/scipyla2017.png)
+
+## Mayo 2017, La Habana, Cuba
+
+[Sitio Oficial ScipyLA 2017](http://scipyla.org/conf/2017/)
+
+### "Más python, más ciencia"
+
+5ta Conferencia Latinoamericana de Computación Cienfítica con Python (SciPyLA 2017), La Habana, Cuba, 10 de Mayo del 2017.
```

Comment:
We can't announce something that's not been negotiated, agreed and signed.
10 de mayo del 2017 => mayo del 2017